massgen/cli.py

```
@@ -460,6 +460,76 @@ def record_event(event):
 )
 
 
+def _is_single_agent_config(config: Dict[str, Any]) -> bool:
```

Ruff (0.15.4)

[error] Undefined name `Dict`

(F821)